### ISSUE TITLE
fullscreen using composite and PAL

### ISSUE THREAD

**Issue by rekotc** [huceke/omxplayer#81]
hello everybody, im having a problem trying to play any full screen video using my old composite connection. The videos play normally but the black bars on the top and bottom of the screen are missing. So i can see the opened windows in the background and the tray bar. Im wondering if this is a problem with the resolution used in PAL system, i have noticed that if i set the raspberry to NTSC output then omxplayer works fine. But in this case i lose part of the vertical resolution of my tv since PAL uses 576 lines and NTSC 480 (in fact i see 2 small black bars added on top and bottom) . Am i the only one with this problem?

**Comment by incyi** [comment]
Same problem here. Picture: https://pbs.twimg.com/media/A_X_7xFCYAEfkeh.jpg:large

Another problem by me is also that the video signal corrupts when you close omxplayer. I get this image:
https://pbs.twimg.com/media/A_YAB4hCUAAMmc5.jpg:large

**Comment by rekotc** [comment]
yes it's exactly what im talking about :), but i dont have the corrupted video signal on exit

**Comment by ghost** [comment]
Same problem here. Unfortunately i can't use NTSC, because my old tv doesn´t support it (if NTSC is enabled output is black/white only)...

**Comment by VladimirPal** [comment]
Same problem:(

**Comment by popcornmix** [comment]
@Cheguevara 
Can you explain exactly the problem you have? Is it you can see background behind window?
